features/step_definitions/registry.rb

```diff
@@ -385,8 +386,18 @@
 end
 
 Given /^default image registry route is stored in the#{OPT_SYM} clipboard$/ do |cb_name| 
+  ensure_admin_tagged
+  ensure_destructive_tagged
   org_proj_name = project(generate: false).name rescue nil
   cb_name ||= :registry_route
   cb[cb_name] = route('default-route', service('default-route',project('openshift-image-registry'))).dns(by: admin)
   project(org_proj_name) if org_proj_name
 end
+
+Given /^current generation number of image-registry deployment is stored into#{OPT_SYM} clipboard$/ do |cb_name|
```

Comment:
Could we make this more suitable for other resources?

Reply:
Good suggestion!

Reply:
Done, please check.